wiki/Building-Windows.md

```diff
@@ -0,0 +1,101 @@
+
+### Setting Up WSL
+
+Return to the command prompt (Win+R, type "cmd" then enter).
+Type `wslconfig /l`, you should have something like [[/img/building-tutorial/badwsl.png]]
+
+You will need to set wsl Ubuntu as the default. Type `wslconfig /s Ubuntu`. You should obtain this after rerunning `wslconfig /l` [[/img/building-tutorial/goodwsl.png]]
+
+### Cloning The Repository and setting up permission
+You are nearly done, you will only need to clone the Hyperspace repository inside wsl to be able to start the build process.
+
+Return to the command prompt.
+Type `wsl`, navigate to the folder you want to clone the Hyperspace repository, then run `git clone https://github.com/FTL-Hyperspace/FTL-Hyperspace`
+[[/img/building-tutorial/clonehs.png]]
+
+This will clone the current Hyperspace repository from GitHub to your computer (when working with Hyperspace it is recommended to fork the Hyperspace repository and to work on it instead)
+
+Now you need to navigate the cloned repository to reach buildscripts/windows/ .sh files, running `cd FTL-Hyperspace/buildscripts/windows/` where you cloned the repository should work (otherwise use `ls` and `cd` to navigate to the right folder)
+
+[[/img/building-tutorial/reachsh.png]]
+
+Inside the folder run `sudo chmod 777 *` to ensure that your system will authorize the execution of those file when building Hyperspace.
+
+### Building Hyperspace
+
+Ensure that you have Docker Desktop running, otherwise the build will not start.
+While still in wsl, inside `FTL-Hyperspace/buildscripts/windows/` run `sudo bash ./build-releaseonly-from-docker.sh`
+
+This will prompt the building of Hypespace. From now on the .sh file will take care of the rest.
+
+The first build of Hyperspace is the lenghty as it has to install all dependencies and build all of Hyperspace, the following one will be faster.
+
+At the end of the building process you should find your new hyperspace.dll at `FTL-Hyperspace\build-windows-release\` ready to be tested.
+
+# Optional
+
+## Hypervisor Error
+During the above guide it may happen that you encounter this error [[/img/building-tutorial/hypererror.png]]
+
+This error means that the hardware visualization is disabled on your motherboard, it is an issue that is hardware specific so we recommend the following solution:
+
+Google "how to enable hardware visualization on `insert motherboard manufacturer here`" and follow the guide that present themselves.
+
+You can find out who's motherboard you have by, hitting Win+R and typing `msinfo32`.
+In the "System Information", the right information board you should find the item "BaseBoard Manufacturer" with its corresponding value.[[/img/building-tutorial/board.png]]
+
+## Building ZHL files (not required to build Hyperspace)
+
+
+For building ZHL files you need some dependencies, run all the following command in `wsl`
+`sudo apt install lua5.3`
+`sudo apt-get install lua5.3-dev`
+`sudo apt-get install lua5.3 luarocks`
```

Comment:
nitpick, be consistent with use of `apt` vs `apt-get`

They're two separate programs but also this can all be combined & be consistent.
apt-get is generally for scripts (because its output is consistent) although an interactive user can use it too, apt is more interactive.
```suggestion
sudo apt install lua5.3 lua5.3-dev luarocks
```
OR
```suggestion
sudo apt-get install lua5.3 lua5.3-dev luarocks
```